### PR TITLE
knex.js library integration

### PR DESCRIPTION
**Pull request by MichaelDurfey**

### CODE REVIEW

File: __tests__/serverSpec.jsx [hunk 1]
```diff
@@ -3,90 +3,67 @@ const app = require('../server/app');
...
-    supertest(app)
-      .post('/')
-      .field('chinese', '105')
-      .expect(201)
-      .end((err) => {
-        if (err) throw err;
-      });
-  });
-});
+// describe('post /', () => {
```
Comment (ZeMunchkin) [inline __tests__/serverSpec.jsx]: You probably don't want to merge commented-out code. Your master should only contain working code

File: db/data/mapData.json [hunk 2]
```diff
@@ -1 +1,1001 @@
...
+  "address": "202 Schamberger Glen North Michele Wisconsin, 97970",
+  "image": "http://lorempixel.com/640/480",
+  "phoneNumber": "845-560-4610",
+  "id": 299
+}, {
+  "address": "049 Skiles Island Smithamport Texas, 18279-7848",
+  "image": "http://lorempixel.com/640/480",
+  "phoneNumber": "991-823-9287",
+  "id": 300
+}]
```
Comment (ZeMunchkin) [inline db/data/mapData.json]: new line

File: db/seeds/restaurant_types.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const restaurantData = require('../data/MomaDummyData.js');
+const titleData = restaurantData.data;
+const { types } = restaurantData;
+/*eslint-disable*/
+
+const newData = titleData.map(t => t.foodType.map(f => ({id_restaurant: t.id, id_types: types[f]}))).reduce((acc, curr) => {
```
Comment (ZeMunchkin) [inline db/seeds/restaurant_types.js]: I feel like this line of code would be more legible if it was spaced onto a couple different lines. Having three HOF in one line is tricky to follow. Did the linter like it better on one line?

**Review by ZeMunchkin — COMMENTED** [review]
See comments

File: db/seeds/restaurant_types.js (hunk 3 above)
Comment (MichaelDurfey) [reply db/seeds/restaurant_types.js]: For sure, good suggestion. Actually the linter is ok with it (sort of). I've fixed it up.

**Pull request merged**